src/hooks/useEtherspotAssets.ts

```diff
@@ -23,16 +23,18 @@ const useEtherspotAssets = (chainId?: number): IEtherspotAssetsHook => {
   }, [chainId, etherspotChainId]);
 
   const getAssets = async (
-    // TODO: use assetsChainId once available on Prime SDK
-    assetsChainId: number = defaultChainId,
+    chainId?: number,
+    name?: string,
```

Comment:
Just wondering if this should have a default name and chain? I'm not sure if the defaults are assumed in the data service itself...

Reply:
For reference, `PillarTokens` is the dafault token